src/vm/wren_opcodes.h

```diff
@@ -7,186 +7,190 @@
-OPCODE(JUMP_IF)
-
-// If the top of the stack is false, jump [arg] forward. Otherwise, pop and
-// continue.
-OPCODE(AND)
-
-// If the top of the stack is non-false, jump [arg] forward. Otherwise, pop
-// and continue.
-OPCODE(OR)
-
-// Pop [a] then [b] and push true if [b] is an instance of [a].
-OPCODE(IS)
-
-// Close the upvalue for the local on the top of the stack, then pop it.
-OPCODE(CLOSE_UPVALUE)
-
-// Exit from the current function and return the value on the top of the
-// stack.
-OPCODE(RETURN)
-
-// Creates a closure for the function stored at [arg] in the constant table.
-//
-// Following the function argument is a number of arguments, two for each
-// upvalue. The first is true if the variable being captured is a local (as
-// opposed to an upvalue), and the second is the index of the local or
-// upvalue being captured.
-//
-// Pushes the created closure.
-OPCODE(CLOSURE)
-
-// Creates a class. Top of stack is the superclass, or `null` if the class
-// inherits Object. Below that is a string for the name of the class. Byte
-// [arg] is the number of fields in the class.
-OPCODE(CLASS)
-
-// Define a method for symbol [arg]. The class receiving the method is popped
-// off the stack, then the function defining the body is popped.
-//
-// If a foreign method is being defined, the "function" will be a string
-// identifying the foreign method. Otherwise, it will be a function or
-// closure.
-OPCODE(METHOD_INSTANCE)
-
-// Define a method for symbol [arg]. The class whose metaclass will receive
-// the method is popped off the stack, then the function defining the body is
-// popped.
-//
-// If a foreign method is being defined, the "function" will be a string
-// identifying the foreign method. Otherwise, it will be a function or
-// closure.
-OPCODE(METHOD_STATIC)
-
-// Load the module whose name is stored in string constant [arg]. Pushes
-// NULL onto the stack. If the module has already been loaded, does nothing
-// else. Otherwise, it creates a fiber to run the desired module and switches
-// to that. When that fiber is done, the current one is resumed.
-OPCODE(LOAD_MODULE)
-
-// Reads a top-level variable from another module. [arg1] is a string
-// constant for the name of the module, and [arg2] is a string constant for
-// the variable name. Pushes the variable if found, or generates a runtime
-// error otherwise.
-OPCODE(IMPORT_VARIABLE)
-
-// This pseudo-instruction indicates the end of the bytecode. It should
-// always be preceded by a `CODE_RETURN`, so is never actually executed.
-OPCODE(END)
\ No newline at end of file
+// The whole file is one big macro, because it is later used inside
+// a '#define' where you cannot use an '#include'.
+#define PLACE_OPCODES                                                           \
+                                                                                \
+/* Load the constant at index [arg]. */                                         \
```

Comment:
Instead of translating this all to `/* */` comments, it's probably simpler to just pull all of the comments out of the macro definition. More like:

```c
// CONSTANT
// Load the constant at index [arg].
//
//
// NULL
// Push null onto the stack.
//
// ...
#define PLACE_OPCODES \
    OPCODE(CONSTANT) \
    OPCODE(NULL) \
    ...
```